go-controller/pkg/ovn/kubevirt.go

```diff
@@ -0,0 +1,73 @@
+package ovn
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kubevirt"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/libovsdbops"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
+	corev1 "k8s.io/api/core/v1"
+	kapi "k8s.io/api/core/v1"
+)
+
+func (oc *DefaultNetworkController) ensureDHCPOptionsForVM(pod *corev1.Pod, lsp *nbdb.LogicalSwitchPort) error {
+	if !kubevirt.AllowPodBridgeNetworkLiveMigration(pod.Annotations) {
+		return nil
```

Comment:
question: should this log something ?

Reply:
Nope, if we do so al log will appear at all the non virt-launcher pods, and clearly we don't want that.